3.1/3.1/main.c

```diff
@@ -0,0 +1,59 @@
+#include <stdio.h>
+
+void swap(int *a, int *b){
+    int tmp = *a;
+    *a = *b;
+    *b = tmp;
+}
+
+void insertionSort(int array[], int low, int high) {
+    for (int i = low + 1; i <= high; i++) {
+        int key = array[i];
+        int j = i - 1;
+        while (j >= low && array[j] > key) {
+            array[j + 1] = array[j];
+            j--;
+        }
+        array[j + 1] = key;
+    }
+}
+
+void qsortCustom(int array[], int low, int high) {
+    if (low < high) {
```

Comment:
```suggestion
    if (low >= high) {
        return;
    }
```
И дальше без лишнего отступа